### PR TITLE
Improve Beacon Node Logging UX

### PR DESCRIPTION
**Pull request by rauljordan**
Part of #3450 

---

# Description

**Write why you are making the changes in this pull request**

This PR does a cleanup of all of our beacon node's logging UX, especially cleaning up useless, redundant, or otherwise too verbose / frequent debug logging.

**Write a summary of the changes you are making**

- [x] Info logs revamped
- [x] Debug logs fixed
- [x] Warn logs revamped

**Link anything that would be helpful or relevant to the reviewers**

### CODE REVIEW

**Comment by terencechain** [comment]
Can you please remove the following as well? Thanks!
` Finished updating validator vote for attestation....`

File: beacon-chain/blockchain/forkchoice/process_block.go [hunk 1]
```diff
@@ -8,6 +8,9 @@ import (
 
 	"github.com/pkg/errors"
 	"github.com/prysmaticlabs/go-ssz"
+	"github.com/sirupsen/logrus"
+	"go.opencensus.io/trace"
+
```
Comment (0xKiwi) [inline beacon-chain/blockchain/forkchoice/process_block.go]: Can you remove this empty line?

File: beacon-chain/blockchain/log.go [hunk 2]
```diff
@@ -1,10 +1,9 @@
 package blockchain
 
 import (
-	"encoding/hex"
+	"github.com/sirupsen/logrus"
```
Comment (0xKiwi) [inline beacon-chain/blockchain/log.go]: Can you fix these imports too?

File: beacon-chain/blockchain/receive_block.go [hunk 3]
```diff
@@ -7,10 +7,11 @@ import (
 
 	"github.com/pkg/errors"
 	"github.com/prysmaticlabs/go-ssz"
-	ethpb "github.com/prysmaticlabs/prysm/proto/eth/v1alpha1"
-	"github.com/prysmaticlabs/prysm/shared/bytesutil"
 	"github.com/sirupsen/logrus"
 	"go.opencensus.io/trace"
+
```
Comment (0xKiwi) [inline beacon-chain/blockchain/receive_block.go]: These too

File: beacon-chain/blockchain/service.go [hunk 4]
```diff
@@ -5,14 +5,16 @@ package blockchain
...
 	"runtime"
 	"sync"
 	"time"
 
 	"github.com/pkg/errors"
-	ssz "github.com/prysmaticlabs/go-ssz"
+	"github.com/prysmaticlabs/go-ssz"
+	"github.com/sirupsen/logrus"
+	"go.opencensus.io/trace"
+
```
Comment (0xKiwi) [inline beacon-chain/blockchain/service.go]: Same here

**Pull request merged**